superset/models/tags.py

```diff
@@ -46,6 +46,7 @@ class TagTypes(enum.Enum):
     can find all their objects by querying for the tag `owner:alice`.
     """
 
+    # pylint: disable=invalid-name
```

Comment:
I opted not the change these enums to uppercase as the string representation of the lowercase name is used throughout and I wanted to minimize the size of the PR.